doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,78 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    	In a multi-image build, this allows you to debug the source code of your application only.
 
 If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
\ No newline at end of file
+If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
+
+.. _gs_programming_cmd:
+
+Building with Command line
+**************************
+
+Complete the following steps to build |NCS| projects with command line after completing the :ref:`command line build environment setup for Windows<build_environment_cli_win>` or the :ref:`command line build environment setup for Linux<build_environment_cli_linux>` for Windows and Linux environments respectively.
+
+1.    Open a terminal window.
+
+#.    Go to the samples or applications directory depending on whether you are building a sample or an application:
+
+      .. code-block:: console
+
+         cd nrf/samples
+         cd nrf/applications
+
+      .. note::
+         All the samples corresponding to the nRF9160 DK board are located at :file:`samples/nrf9160`.
+
+#.    Build the sample or application using the west command.
+      The development board is specified by the parameter ``board_name``, and the sample or application directory is specified by the parameter ``directory_name`` in the west command as follows:
```

Comment:
Variables (like board_name and directory_name) should be in italics: *board_name* (both here where they are introduced and in the code block).
This makes it clearer that they must be replaced in the command.

To make the italics work in the code block below, replace ".. code-block:: console" with:
   .. parsed-literal::
&nbsp;&nbsp;      :class: highlight

Reply:
Updated.